static/css/style.css

```diff
@@ -506,6 +506,21 @@ a.command:hover {
     width: 200px;
 }
 
+.snippet-button {
+    text-decoration: none;
+    background: none;
+    color: inherit;
+    border: none;
+    border-radius: 5px;
+    padding: 2px;
+    font: inherit;
+    cursor: pointer;
+    outline: inherit;
+}
+.snippet-button:focus {
+  box-shadow: 0px 0px 5px 3px #FFCC33;
```

Comment:
We will need a PR in python-editor for the foundation colours, could you prepare that as well?

Reply:
If `FFCC33` is the upstream yellow we will need a new branch from python-editor overwriting in style-foundation.css the new classes setting this colour to the foundation yellow.
This needs to be a new branch from the python-editor master, as it's in a different file not present here.